README.md

```diff
@@ -19,6 +19,7 @@ To install the plugin, run:
 - `--namespace` | `-n`: the Kubernetes namespace in which the CephCluster resides (default: rook-ceph)
 - `--operator-namespace` | `-o`: the Kubernetes namespace in which the rook operator resides (default: rook-ceph)
 - `--help` | `-h`: Output help text
+- `--context=<current_context>`: the name of the context to be used
```

Comment:
Let's put this one line higher so the --help option remains last in the list
```suggestion
- `--context=<context_name>`: the name of the Kubernetes context to be used
```

Reply:
done